README.md

```diff
@@ -1113,7 +1113,7 @@ Table of Contents
    * [SeaTable](https://seatable.io/) — Flexible, Spreadsheet-like Database built by Seafile team. unlimited tables, 2,000 lines, 1-month versioning, up to 25 team members.
    * [skyvia.com](https://skyvia.com/) — Cloud Data Platform, offers free tier and all plans are completely free while in beta
    * [StackBy](https://stackby.com/) — One tool that brings together flexibility of spreadsheets, power of databases and built-in integrations with your favorite business apps. Free plan includes unlimited users, 10 stacks, 2GB attachment per stack.
-   * [TiDB Cloud](https://tidbcloud.com/signup) — TiDB is an open source MySQL compatible distributed HTAP RDBMS. TiDB Cloud provides one-year free Developer Tier, 10 GiB of OLTP storage, and 10 GiB of OLAP Storage.
+   * [TiDB Cloud](https://tidbcloud.com/free-trial) — TiDB is an open source MySQL compatible distributed HTAP RDBMS. TiDB Cloud provides one-year free Developer Tier, 500 MiB of OLTP storage, and 500 MiB of OLAP Storage.
```

Comment:
https://en.pingcap.com/tidb-cloud/ is a better address for this, linking to just the login screen does not help people review what this is about

Reply:
Thanks for pointing out, let's change it to this informative landing page.